exercises/practice/gigasecond/.meta/example.c

```diff
@@ -1,6 +1,10 @@
 #include "gigasecond.h"
 
-time_t gigasecond_after(time_t start)
+#define GIGASECOND 1000000000
+
+void gigasecond(time_t start, char *buffer, size_t size)
 {
-   return start + 1000000000;
+   time_t after = start + GIGASECOND;
+   struct tm *t = gmtime(&after);
```

Comment:
The implementation changes look good to me.
I _think_ this usage does not run afoul of #319 but maybe @ryanplusplus can double-check?

Reply:
I _think_ it's still okay because of the fix we have in place.